IntroToCSharp/Pages/Lessons/Variables/TheTypeSystem.razor

```diff
@@ -0,0 +1,141 @@
+@inherits CaptainCoder.Page
+@page "/lessons/variables/the-type-system"
+
+<Title>The C# Type System</Title>
+
+<Section Title="Introduction">
+    <MudText>
+        In the <MudLink Href="@BasicDataTypes.Href">Basic Data Types</MudLink>
+        lesson, we learn that all
+        <Keyword Word="data" /> in a computer is stored
+        as sequences of
+        <Keyword Word="bits" />. As programmers, we design
+        different
+        <Keyword Word="binary encoding" />s to represent different
+        types of data. These are called
+        <Keyword Word="data type" />s. C# has
+        <b>14</b> built in
+        <Keyword Word="primitive data type" />s that we use to
+        encode different types of data. This section of the book is a high level
+        reference of those 14
+        <Keyword Word="data type" />s.
+    </MudText>
+</Section>
+<Section Title="Integers">
+    <MudText>
+        There are 8 different integer types each of which is represented using a
+        different number of <Keyword Word="byte" />s as well as being <Keyword Word="signed"/> or
+        <Keyword Word="unsigned"/>.
+    </MudText>
+    <Note>You do not need to memorize this table but it is good to be aware.</Note>
+    <DataTable Headers="@intHeaders" Rows="@intRows"></DataTable>
+</Section>
+
+<Section Title="Characters">
+    <MudText>
+        In C#, the <Keyword Word="char"/> type represents a single character. This
```

Comment:
This is pretty wordy and gets a little confusing reading it.

Reply:
How about this?